python/flashinfer/jit/batch_decode_mla_templ.py

```diff
@@ -83,8 +84,9 @@
   auto device = q_nope.device();
   int64_t batch_size = q_nope.size(0);
   int64_t num_qo_heads = q_nope.size(1);
-  int64_t page_size = paged_ckv_cache.size(1);;
+  int64_t page_size = paged_ckv_cache.size(1);
 
```

Comment:
Although this is a code template, I think there's an extra semicolon here， delete it btw